src/helpers/axios.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
```

Comment:
rename axios.js file to this axiosHelper.js to have the same naming convention with others

Reply:
Thanks man